src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt

```diff
@@ -38,48 +41,20 @@ class MavenPublishPluginIntegrationTest(
   @get:Rule val testProjectDir: TemporaryFolder = TemporaryFolder()
 
   private lateinit var repoFolder: File
-  private lateinit var buildFile: File
   private lateinit var artifactFolder: String
 
   @Before fun setUp() {
     repoFolder = testProjectDir.newFolder("repo")
-    buildFile = testProjectDir.newFile("build.gradle")
-    buildFile.writeText("""
-        plugins {
-          id "com.vanniktech.maven.publish"
-        }
-
-        repositories {
-            google()
-            mavenCentral()
-            jcenter()
-        }
-
-        mavenPublish {
-          useMavenPublish = $useMavenPublish
-          targets {
-            installArchives {
-              releaseRepositoryUrl = "file://${repoFolder.absolutePath}"
-              signing = true
-            }
-            uploadArchives {
-              releaseRepositoryUrl = "file://${repoFolder.absolutePath}"
-              signing = true
-            }
-          }
-        }
-        """)
 
-    testProjectDir.newFile("gradle.properties").writeText("""
+    File("$FIXTURES/common").listFiles()?.forEach { it.copyRecursively(File(testProjectDir.root, it.name)) }
```

Comment:
super nit: we could `testProjectDir.root.resolve(it.name)` feel free to do in another PR

Reply:
I didn't know about resolve, nice 
43031c2c35292e7f1b67dbba167411acd9f8e0d1